spyderlib/plugins/ipythonconsole.py

```diff
@@ -177,12 +173,22 @@ def setup_page(self):
 
         interface_layout = QVBoxLayout()
         interface_layout.addWidget(banner_box)
-        interface_layout.addWidget(gui_comp_box)
         interface_layout.addWidget(pager_box)
         interface_layout.addWidget(calltips_box)
         interface_layout.addWidget(ask_box)
         interface_group.setLayout(interface_layout)
 
+        comp_group = QGroupBox(_("Completion Widget"))

```

Comment:
Because of the text below, this should be called *Completion type*, instead of *Completion widget*.

I mean, the completion widget is only one type of completion